src/_P128_NeoPixelBusFX.ino

```diff
@@ -202,8 +205,14 @@ boolean Plugin_128(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_INIT:
     {
-      initPluginTaskData(event->TaskIndex,
-                         new (std::nothrow) P128_data_struct(PIN(0), PCONFIG(0)));
+      // *Ensure* there is no currently running process, if so, just clear it out neatly
+      P128_data_struct *P128_clear = static_cast<P128_data_struct *>(getPluginTaskData(event->TaskIndex));
+
+      if (nullptr != P128_clear) {
+        clearPluginTaskData(event->TaskIndex);
```

Comment:
Just wondering, how can this happen?
There should have been a call to PLUGIN_EXIT before PLUGIN_INIT.

Reply:
When I ran `taskenable` on the task while still running (on ESP32s2) it fell into some loop that RMT (IR related [Remote Control](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/rmt.html) stuff, didn't read all the details, but it is an ESP32 feature) was already in use. Solution: add an extra check & clean.
(_Could_ be some lingering but, I put on the backburner for now.)

Reply:
The `PLUGIN_INIT` event doesn't call `PLUGIN_EXIT` in the internal flow, so I'll leave the check in place, for now.

The library used for the effects does automatically try to use the next internal RMT interface, but won't accept that on the same pin, that is most likely the cause of the error I mentioned. Creating a new object 'on top' of an existing object, doesn't destroy the initial object, but now that I actively destroy it, all is de-initialized/cleaned up as required.

If calling `PLUGIN_EXIT` when the task is still active and the task is being activated again, is the expected/intended behavior, then we should probably fix that in a separate PR.